lib/smart_answer_flows/marriage-abroad/outcomes/os_affirmation/_fees.govspeak.erb

```diff
@@ -4,7 +4,10 @@
 
 <% unless calculator.country_without_consular_facilities? ||
    calculator.ceremony_country == 'indonesia' ||
-   calculator.ceremony_country == 'monaco' %>
+   calculator.ceremony_country == 'monaco' ||
+   calculator.ceremony_country == 'france' ||
+   calculator.ceremony_country == 'new-caledonia' ||
+   calculator.ceremony_country == 'wallis-and-futuna' %>
```

Comment:
Minor: I wonder whether there would be any benefit of grouping `france`, `new-caledonia` & `wallis-and-futana` together somehow (either by parenthesising the conditions, or even extracting a method onto the calculator) to retain the concept of "France and its overseas territories" which was implied by the name of the outcome.